tests/unit/compareUserInListTest.ts

```diff
@@ -0,0 +1,24 @@
+import {compareUserInList} from '@pages/home/report/ReportActionCompose/SuggestionMention';
+
+describe('compareUserInList', () => {
+    it('Should compare the weight if the weight is different', () => {
+        const first = {login: 'John Doe', weight: 1, accountID: 1};
+        const second = {login: 'Jane Doe', weight: 2, accountID: 2};
+        expect(compareUserInList(first, second)).toBe(-1);
+    });
+
+    it('Should compare the displayName if the weight is the same', () => {
+        const first = {login: 'águero', weight: 2, accountID: 3};
+        const second = {login: 'Bronn', weight: 2, accountID: 4};
+        const third = {login: 'Carol', weight: 2, accountID: 5};
+        expect(compareUserInList(first, second)).toBe(-1);
```

Comment:
Check the other cases too (eg, `expect(compareUserInList(second, first)).toBe....` )